### ISSUE TITLE
adapter updates

### ISSUE THREAD

**Issue by DougAnderson444** [wighawag/sveltejs-adapter-ipfs#4]
Hello!  I noticed after the recent svelteKit adapter overhaul (https://github.com/sveltejs/kit/pull/2931) that this adapter kind of breaks on a rimraf issue.

I don't uderstand the inner workings of the adapter to comment further, just thought I'd leave this here as I've been using this adapter until now with great success, thanks :)

```
19:44:03.223	> Using sveltejs-adapter-ipfs
19:44:03.225	> Cannot read property 'rimraf' of undefined
19:44:03.225	TypeError: Cannot read property 'rimraf' of undefined
19:44:03.226	    at adapt (file:///opt/buildhome/repo/node_modules/sveltejs-adapter-ipfs/index.js:9:10)
19:44:03.226	    at adapt (file:///opt/buildhome/repo/node_modules/@sveltejs/kit/dist/chunks/index5.js:623:8)
19:44:03.226	    at file:///opt/buildhome/repo/node_modules/@sveltejs/kit/dist/cli.js:948:11
19:44:03.245	Failed: build command exited with code: 1
```

**Comment by wighawag** [comment]
Hi @DougAnderson444 thanks for flagging this, I ll have a look.

**Comment by wighawag** [comment]
Did not tested much but I made a fix in 0.2.0

**Comment by DougAnderson444** [comment]
Thanks for having a look, i appreciate it. 

For anyone else who's watching, the last working svelte kit version that I saw working with this version was 188
